packages/ui-extensions/src/surfaces/admin/components/Select/examples/basic-select.example.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  extension,
+  Select,
+} from '@shopify/ui-extensions/admin';
+
+export default extension(
+  'Playground',
+  (root) => {
+    const select = root.createComponent(Select, {
+      label: 'Country',
+      value: '2',
```

Comment:
Should we make this example more complete by setting the `onChange` and `value`?

Reply:
I think it's important to have both examples. This is uncontrolled, but we should also have a controlled input example.

Reply:
I don't think uncontrolled inputs actually work in remote UI. You need to update the value after `onChange` otherwise it just reverts back to the original value